### PR TITLE
Add Node.js example and refresh .Net dependencies

### PR DESCRIPTION
**Pull request by archive**
## Why
We want to provide updated and working sample code for creating shipments via Urb-it Delivery API.

## What
1. Removed the old v3 samples since that API has been deprecated
2. Move all v4 samples to the repository root
3. Updated all .Net dependencies
4. Added Node.js sample code

## Screens
image

### CODE REVIEW

File: nodejs/app.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  console.log("\nGet shipping label 🏷️");
+  await getShippingLabel(authorization, trackingNumber);
+  console.log(">> Done! 🌟");
+  console.log(">> zpl saved in current folder");
+
+  console.log("\nBye bye");
+};
+
+(async () => {
```
Comment (niklas-r) [inline nodejs/app.js:25]: Top level await is available from Node v14 so you don't need to wrap this in an async IFEE

**Review by niklas-r — APPROVED** [review]
👍

File: nodejs/labels.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import fetch from "node-fetch";
+import fs from "fs";
+
+const getShippingLabel = async (authorization, trackingNumber) => {
```
[the remark this answers is not in the document]
Comment (archive) [reply nodejs/labels.js:4]: Agree!

File: nodejs/app.js (hunk 1 above)
Comment (archive) [reply nodejs/app.js:25]: Of course, let's change.

**Pull request merged**